### DISCUSSION TITLE
Full Metal Theme

### DISCUSSION THREAD

**Discussion by BlackHargaan** [mathoudebine/turing-smart-screen-python#432] [Themes]
Here is my first theme. No yaml file because I added a weather API and code to get both my ssds to show up. red is 255, 46, 40 and 234, 84, 80.

[fullmetal.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/13803816/fullmetal.zip)

![2](https://github.com/mathoudebine/turing-smart-screen-python/assets/96963662/50403433-667a-4113-88c6-f974bdb8081a)
![3](https://github.com/mathoudebine/turing-smart-screen-python/assets/96963662/f67bd8d3-1bc8-48f8-810e-a075bfd1d921)
![4](https://github.com/mathoudebine/turing-smart-screen-python/assets/96963662/c5aa853e-98ff-4b48-820b-b4319e42fff0)
![1](https://github.com/mathoudebine/turing-smart-screen-python/assets/96963662/e59c83f5-c79b-4823-9806-7ff456744b93)

**Comment by Rikostan** [comment]
I would love a quick and dirty tutorial on how you got the weather data. I do have my own station and access to the data via an API

**Reply by AlexIonut8** [reply]
in which software can i create a theme ?
